ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs

```diff
@@ -405,9 +409,7 @@ storeLedgerStateAt slotNo ledgerAppMode doChecksum env = do
           storeLedgerState oldLedger
           pure (Stop, oldLedger)
 
-    tickThenXApply = case ledgerAppMode of
-        LedgerReapply -> pure ..: tickThenReapply
-        LedgerApply   -> tickThenApply
+    tickThenXApply = tickThenApply appOpts
```

Comment:
As in my other comment: I'd much prefer this code to stay the way it was instead of polluting the core `ouroboros-consensus` code with that flag.

Or am I overlooking something?